bin/start

```diff
@@ -0,0 +1,77 @@
+# vi:ft=sh
+#
+# Version: 20170707-20:21
+
+set -euC
+
+cd "$root"
+
+# Load settings
+[ -f "$root/bin/settings" ] && . "$root/bin/settings"
+
+###
+# Stuff to set in settings file.
+
+# Codecov token for coverage reports; only needed for private repos.
+codecov_token=${codecov_token:-}
+
+# Tags to add to "go test" when running tests.
+test_tags=${test_tags:-}
+
+# Verbose output?
+verbose=${verbose:-0}
+
+
+###
+# For the following the defaults should almost always be okay. If not, then
+# consider fixing the project maybe?
+
+# Binary name.
```

Comment:
Actually the way it's used "Docker image name" is more accurate:

    docker build -t "teamwork/$name:$docker_tag" .
    docker push "teamwork/$name:$docker_tag"